integration/integration_test.go

```diff
@@ -459,10 +459,6 @@ func TestSeccompadvisor(t *testing.T) {
 
 	t.Parallel()
 
-	if *githubCI {
-		t.Skip("seccomp-advisor timed out within GitHub CI.")
-	}
-
```

Comment:
Why is this part of this PR?

Reply:
Before this commit, `seccomp-advisor` would timeout in the CI (I think it is also the same for `*-collector`) due to the fact we were rewritting the operation before waiting its end.
So, with the previous commit of this PR, we can enable it (and maybe `*-collector`, I need to check to be sure).

Reply:
I don't think collectors suffer the same issue because they have only one operation. I think they cannot run on GitHub because the GitHub runners use a kernel that doesn't support them. Could you please confirm it and add a comment that explains it?

Reply:
I was because GitHub CI is lacking BTF.
But I think now it should be OK with `btfgen`, I do not remember exactly the order but I think I wrote the CI tests before `btfgen` was available for it.